src/raster/tests.rs

```diff
@@ -759,17 +734,17 @@ fn test_raster_stats() {
     assert_eq!(
         rb.get_statistics(true, false).unwrap().unwrap(),
         StatisticsAll {
-            min: 12.0,
+            min: 0.0,
             max: 255.0,
-            mean: 89.2526,
-            std_dev: 90.99835379412092,
+            mean: 68.4716,
+            std_dev: 83.68444773934999,
```

Comment:
Any idea why the statistics changed? Is it computing approx stats via overviews?

Reply:
No, the original TIFF was somehow different from the PNG:

![image](https://github.com/georust/gdal/assets/308347/c138c975-6822-4ad4-94bc-f83f2025143e)

Reversed, on top, is the original TIFF, on bottom is the PNG converted to TIFF. Notice the darker area at the South pole and what looks like JPEG artifacts (but might be something else). I think both versions are derived from the real Blue Marble, but perhaps not in an ideal way.